front/src/applications/stdcm/getStdcmTimetable.jsx

```diff
@@ -27,19 +24,19 @@ export default async function getStdcmTimetable(
     if (!simulation.trains || !simulation.trains[selectedTrain]) {
       dispatch(updateSelectedTrainId(undefined));
     }
-    const timetable = await get(`${timetableURI}${timetableID}/`);
+    const timetable = await get(`/editoast/timetable/${timetableID}/`);
```

Comment:
there is no editoast endpoint defined in rtk?

Reply:
As getStdcmTimetable is not a component, I can't use hooks here :/ That's why I only changed the url (and the params where needed), but we will have to refacto this later, when we will migrate this file to ts

Reply:
you can use rtk endpoints without hooks, but it’s messy.
But here you can just turn getStdcmTimetable into a custom hook to call hooks inside.

Reply:
This file is not useful in the end, so I'm just deleting it :)